build.rs

```diff
@@ -0,0 +1,73 @@
+
+extern crate build_const;
+
+pub fn make_table_crc32(poly: u32) -> [u32; 256] {
```

Comment:
We could move them into an internal "crate" which both `build.rs` and the individual modules depend on. I can do that if you would like.

I also didn't like copy/pasting them.